util/testutil.h

```diff
@@ -231,6 +231,47 @@ class StringSink: public WritableFile {
   size_t last_flush_;
 };
 
+class OverwritingStringSink : public WritableFile {
+ public:
+  std::string contents_;
```

Comment:
Minor: Why is this public ? Please use contents() function to access it if required.